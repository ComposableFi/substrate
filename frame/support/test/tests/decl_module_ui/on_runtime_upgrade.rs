frame_support::decl_module! {
	pub struct Module<T: Config> for enum Call where origin: T::Origin, system=self {
		fn on_runtime_upgrade() -> Weight { 0 }
	}
}

fn main() {}
