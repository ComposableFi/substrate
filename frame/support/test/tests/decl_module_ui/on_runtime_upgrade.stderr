error: `on_runtime_upgrade` can't be implemented anymore. The runtime upgrade must be defined at the runtime level.
 --> tests/decl_module_ui/on_runtime_upgrade.rs:1:1
  |
1 | / frame_support::decl_module! {
2 | |     pub struct Module<T: Config> for enum Call where origin: T::Origin, system=self {
3 | |         fn on_runtime_upgrade() -> Weight { 0 }
4 | |     }
5 | | }
  | |_^
  |
  = note: this error originates in the macro `$crate::decl_module` (in Nightly builds, run with -Z macro-backtrace for more info)
